tools/bpf_inspect.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env drgn
+# SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
```

Comment:
I didn't notice this last time, but drgn is GPL-3.0+. If we have tools under different licenses, it might complicate things if we want to lift common code into helpers. Would you be okay with licensing this as GPL?